src/tools/opensc-explorer.c

```diff
@@ -2176,6 +2177,10 @@ static int do_asn1(int argc, char **argv)
 		}
 	}
 
+	/* workaround when the issuer of a card does prefix the EF.ATR payload with 0x80 */
+	if ((buf[0] == ISO7816_II_CATEGORY_TLV) &&
+	    (memcmp(path.value, "\x3f\x00\x2f\x01", 4) == 0))
```

Comment:
Checking for the path of EF.ATR is a good workaround. just one more corner case: if the user specified an offset, then we should not automatically increment it